Library/Formula/rabbitmq.rb

```diff
@@ -1,14 +1,10 @@
-# We'd really like this formula to be built from source so we can build
-# against a modern OpenSSL rather than the prehistoric system version.
-# Submit a PR if you're interested, Thanks!
 class Rabbitmq < Formula
   desc "Messaging broker"
   homepage "https://www.rabbitmq.com"
-  url "https://www.rabbitmq.com/releases/rabbitmq-server/v3.6.0/rabbitmq-server-mac-standalone-3.6.0.tar.xz"
-  sha256 "db10cb920cfc77f5714ca92275dcb5d4870301817debcc27c399041a4ed87ea8"
-
-  bottle :unneeded
```

Comment:
Nope; removing `bottle :unneeded` was wrong. I'll revert that change.

(I had been thinking that Homebrew now needed to “embed” the Erlang in the style that the official Mac distribution had been doing, but now I see that was just late-night delirium)